packages/edit-site/src/components/post-edit/index.js

```diff
@@ -78,6 +67,43 @@ function PostEditForm( { postType, postId } ) {
 		'comment_status',
 	];
 
+	const form = useMemo(
+		() => ( {
+			type: 'panel',
+			fields: [
+				'featured_media',
+				'title',
+				'status_and_visibility',
+				'author',
+				'date',
+				'slug',
+				'comment_status',
+			].filter(
+				( field ) =>
+					ids.length === 1 ||
+					fieldsWithBulkEditSupport.includes( field )
+			),
+			combinedFields: [
+				{
+					id: 'status_and_visibility',
+					label: __( 'Status & Visibility' ),
+					children: [ 'status', 'password' ].filter( ( child ) => {
```

Comment:
The API for controlling [visibility](https://github.com/WordPress/gutenberg/pull/65826/) landed, but it didn't have support for combinedFields. We also have https://github.com/WordPress/gutenberg/pull/66531 that aims to replace combinedFields API.

How would you like to move forward? Do we merge 66531 first and come back to this one later? Implement visibility for combinedFields as it is here? Any other approach?

Reply:
Given this is ready, I would say merge this first, and replace the `combinedFields` usage as part of https://github.com/WordPress/gutenberg/pull/66531
I also rebased this PR and made use of the `isVisible` function.